src/shared/components/myPage/ReviewModals.tsx

```
@@ -45,63 +61,134 @@ export const ShareModal = ({ isOpen, onClose, restId }: ModalProps) => {
+          onClick={() => fileSelector.current?.click()}
+        >
+          <img src={camera} />
+          사진 첨부하기
+        </button>
+      ) : (
+        <div className="flex max-w-[300px] items-center gap-2 overflow-x-auto">
+          {previewUrls.map((url, idx) => (
+            <div className="relative shrink-0 rounded-lg">
+              <img src={url} className="aspect-square w-[80px] rounded-lg" />
+              <span
+                className="absolute right-1 top-0 cursor-pointer"
```

Biome (2.3.14)

[error] Missing key property for this element in iterable.

The order of the items may change, and having a key can help React identify which item was moved.
Check the React documentation. 

(lint/correctness/useJsxKeyInIterable)